Sources/BlogModule/Controllers/BlogPostController.swift

```diff
@@ -68,34 +68,22 @@ struct BlogPostController: FeatherController {
 
 /// Overide default Route Builder
 extension BlogPostController {
-    
-    func listApiTimestamp(_ req: Request) throws -> EventLoopFuture<Response> {
-        let start: Date = req.query["start"] ?? Date(timeIntervalSince1970: 0)
-        let end: Date = req.query["end"] ?? Date()
-        let paginated =  listLoader.paginate(req, start: start, end: end, withDeleted: true).map { pc -> PaginationContainer<ListApi.ListObject> in
+
```

Comment:
@tib Here I cleanup the code, didn't like as there was to much. So not I am simply overriding `func listPublicApi(` 
I couldn't figure out a way to actually put that in the model, I tried to have a generic Filter called from any API, that would be call by QueryBuilder, with a default extension just returning ```qb``` and have  ```\.$updatedAt >= req.query["start"]``` overriden in the give filter only in some model. If I get a chance I'll try again. For now, at least we have less code and it provide an example on how to implement custom params for specific API route. 
                     ?? Date(timeIntervalSince1970: 0))``` implemented only in the

Reply:
Ah I see, don't worry too much about it, we'll find a better solution when we see more & more use cases. ;)